.github/workflows/build_test.yml

```diff
@@ -30,6 +30,25 @@ jobs:
       - name: Run fix_lint
         run: python -m tox -e fix_lint
 
+  rstcheck:
+    name: Check rst files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Display Python version
+        run: python -c "import sys; import os; print(\"\n\".join(os.environ[\"PATH\"].split(os.pathsep))); print(sys.version); print(sys.executable);"
+      - name: Upgrade setuptools and pip
+        run: python -m pip install -U setuptools pip
+      - name: Install rstcheck
+        run: python -m pip install rstcheck
+      - name: Run rstcheck
+        run: rstcheck *.rst
+
+
```

Comment:
This is not necessary, because by adding it to precommit-hooks it is already part of the `fix_lint` merge gate.
You can see this on your purposefully wrong commit's action run: https://github.com/snowflakedb/snowflake-connector-python/runs/1265701606

On top of being a merge gate having it as part of our precommit-hooks makes this check runnable locally on dev machines.

Reply:
I removed this merge gate